pcsx2/gui/Dialogs/CreateMemoryCardDialog.cpp

```diff
@@ -252,8 +252,12 @@ void Dialogs::CreateMemoryCardDialog::CreateControls()
 		.	SetToolTip(_t("Use at your own risk.  Erratic memory card behavior is possible (though unlikely)."))
 		.	SetInt(64),
 
-		RadioPanelItem(_("Folder [experimental]"), _("Store memory card contents in the host filesystem instead of a file."))
-		.	SetToolTip(_t("Automatically manages memory card contents so that the console only sees files related to the currently running software. Allows you to drag-and-drop files in and out of the memory card with your standard file explorer. This is still experimental, so use at your own risk!"))
+		RadioPanelItem(_("Folder [best with PCSX2]"), _("Transforms the memory card into an usable folder with files inside."))
+		.	SetToolTip(_t("Gives every game 8 MB for the saves (essentially a whole memcard).\n" 
+						  "You can see the structure and the saves with your File Explorer.\n"
+						  "Can be used to back-up individual saves instead of all saves on a memcard.\n" 
+						  "Incompatible with mymc(plus) which is a save manager.\n"
```

Comment:
I do hesitate to list off external programs by name but if necessary (I feel it kind of is because MyMC is a bit of a lesser known tool), I would consider "Incompatible with PS2 memory card editing tools (e.g. MyMC, MyMCPlus)". This way it says the tools as a general concept are incompatible, not just MyMC.